### DISCUSSION TITLE
why reconnecting (failed to dial ngrok server with address "connect.ngrok-agent.com:443": dial tcp: lookup connect.ngrok-agent.com: i/o timeout)

### DISCUSSION THREAD

**Discussion by ujikstark** [ngrok/ngrok#32] [Bug Reports]
![Uploading ngrok.JPG…]()

**Comment by russorat** [comment]
Hi @ujikstark, thank you for the discussion. i can't get the image to load, but usually when we see that error in the agent, it is because some firewall is blocking access to connect.ngrok-agent.com, which is how our agents connect to our cloud. Is there any way for you to check?

also, you can try running `ngrok diagnose` to see if it provides any additional details.

**Comment by russorat** [comment]
Hi, I haven't heard back from you, so i'm going to close this for now. If you're still having issues, please feel free to open a new discussion with the details.
